content/zh/docs/tutorials/configuration/configure-redis-using-configmap.md

```diff
@@ -79,7 +79,7 @@ Apply the ConfigMap created above, along with a Redis pod manifest:
 
 ```shell
 kubectl apply -f example-redis-config.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/pods/config/redis-pod.yaml
```

Comment:
This URL is strange. Should use `https://k8s.io/examples/pods/config/redis-pod.yaml`.